src/creds.jl

```diff
@@ -36,3 +48,16 @@ mutable struct ClientCredentials <: Credentials
     ClientCredentials(client_id, client_secret, client_credentials_url = nothing) =
         new(client_id, client_secret, client_credentials_url, nothing)
 end
```

Comment:
Can we add a note to the docstring for this struct explaining that the client_secret is hidden by default when displaying this object, and guiding them towards how to actually print the secret if they want to?

They could always do `ctx.credentials.client_secret`, and that will show it, but maybe they wouldn't know how to do that? 🤔 

I dunno, i can't shake the feeling that someone might actually _want_ to print their credentials, and won't be able to figure out how to.